lib/Conversion/ImportVerilog/Structure.cpp

```diff
@@ -82,6 +159,10 @@ struct MemberVisitor {
       initial = context.convertExpression(*init);
       if (!initial)
         return failure();
+
+      if (initial.getType() != type) {
```

Comment:
As follow.

Reply:
Very nice! Can you create an upstream PR for these with a corresponding test? Like the `bit a = 1'b1` that you've mentioned. The test should be failing without your change to `Structure.cpp`, and then work with your change.

Reply:
Sure! I'll create another PR upstream with a test case.